### ISSUE TITLE
Update doc strings

### ISSUE THREAD

**Issue by jlaura** [DOI-USGS/usgscsm#453]
The autogenerated sphinx docs are terrific! There is work to be done now ensuring that all methods are documented and that the docs are cleaned. Example of that cleaning include removing instructions to research something and `???` instances for parameters, among others.

### Library Reference

- [ ] USGSCSM Distortion Class
- [ ] USGSCSM Utilities Class
- [ ] USGSCSM Astro Plugin Class
- [ ] USGSCSM Astro Frame Sensor Model Class
- [ ] USGSCSM Linescan Sensor Model Class
- [ ] USGSCSM SAR Sensor Model Class

### Cameras
It appears that only a subset of the cameras that can be used with USGSCSM are rendering examples. This issue completed would be an up to date example of instantiating a sensor model for each (longer term, I anticipate moving this to one or more pages in a /examples/<> section of the documentation.

- [ ] All camera examples updated and rendering

### Creating an environment
These look like installation instructions. If so, we should move these first and fix the rendering.

- [ ] Install instructions shifted to the top and updated to fix all rendering. 
- [ ] If installation with `pip` is supported add that.

### Creating and testing an ISD
I think we probably should remove the `ale` how to and link to those docs once released. Then we can assume one has an ISD already and go from there. A great first tutorial (FY24) would be getting the USGSCSM stack installed, creating an ISD, and then computing something (footprint, photometric angles on a DEM, etc.)

**Comment by antonhibl** [comment]
@jlaura I am working on the updating of camera rendering examples and wondering if there is somewhere I can find examples of the rendering(s) for each camera model available through CSM to use or if I will need to develop some examples for these docs?

**Comment by Kelvinrr** [comment]
It seems @jlaura is talking about creating copy-pasta style examples for instantiating a camera model for more instruments? Similar to https://usgscsm.readthedocs.io/en/latest/sensors/Cassini.html. If so, I think that belongs more in the `asc-public-docs` since that is a combination of ALE and USGSCSM code in the examples.

**Comment by jlaura** [comment]
Yup, these might be two examples.  Create an ALE ISD and create and ALE state. Then these should 'just' work for all supported sensors, right? The docs could link to ALE documentation that shows which sensors are supported. 

Like @Kelvinrr said, these should be in the ASC docs. Not sure why I included that here. Maybe this predated those docs?
